fera_starter.py

```
@@ -1,3 +1,27 @@
+    #MAC = (platform.system() == "Darwin")
+    if(WINDOWS):
+        exe_path = os.path.join(os.getcwd(), 'FERA', 'FERA_WINDOWS', 'fera.exe')
+    elif(LINUX):
+        exe_path = os.path.join(os.getcwd(), 'FERA', 'FERA_LINUX', 'fera')
+    
+    pathdb = None
+    for filename in os.listdir(os.path.join(os.getcwd(), 'FERA')):
+        if(os.path.isdir(filename)):
+            continue
+        filenamenoext, extension = os.path.splitext(filename)
+        if ("fera" in filenamenoext.lower() and ".db" == extension.lower()):
```

Ruff

Yoda condition detected

Rewrite as `extension.lower() == ".db"`

(SIM300)